Mittons.Fixtures/Docker/Attributes/HealthCheckAttribute.cs

```diff
@@ -11,13 +11,13 @@ public class HealthCheckAttribute : Attribute, IOptionAttribute
 
         public string Command { get; set; }
 
-        public TimeSpan? Interval { get; set; }
+        public int Interval { get; set; }
```

Comment:
Any benefits in using unsigned int for these? Seeing that they're only utilized if they're greater than 0.

Reply:
That's a good point...

I actually changed these to bytes, so we're getting unsigned, but we're also limiting size. Given the nature of these flags, if you need something larger than a byte, you likely are doing something wrong. If your service can't spin up within 18 hours, you have a big problem...